index.js

```diff
@@ -91,33 +91,56 @@ const stonksifyRules = rules.sort(function compare(a, b) {
   }
 });
 
-input.on("keyup", function (e) {
+input.on("keyup", function(e) {
   if (e.which === 13) {
     console.log("Hit Enter");
     enterWord(input.val());
   }
 });
 
-submitButton.on("click", function () {
+submitButton.on("click", function() {
   console.log("Clicked Submit");
   enterWord(input.val());
 });
 
-shareButton.on("click", function () {
+shareButton.on("click", function() {
   copyToClipboard();
-  shareButton.text("Copied to Clipboard!");
+  // adds a span to display "Link Copied"
+  // then, styles the text above the input field's position and animates the text to float upward before removing from the dom
+  input.after('<span id="copied_popup">Link Copied!</span>');
+  let copiedTop = input.offset().top - input.height() - 5;
+  let copiedLeft = input.offset().left + 40;
+  $("#copied_popup")
+    .css("position", "absolute")
+    .css("display", "inline")
+    .css("padding", "0")
+    .css("margin", "0")
+    .css("color", "red")
+    .offset({
+      top: copiedTop,
+      left: copiedLeft
+    })
+    .animate({ top: "-=15" }, 400, "swing");
+
+  let copiedPopup = $("#copied_popup");
```

Comment:
Why is this defined down here? $("#copied_popup") is used on 113.

Reply:
has a bunch of function calls and doesn't return the jquery object

Reply:
after moving the calls to css the object is returned and I can do your suggestion